components/CMakeLists.txt

```diff
@@ -28,75 +27,47 @@ if (cisst_FOUND_AS_REQUIRED)
   include (${CISST_USE_FILE})
 
   # catkin/ROS paths
-  cisst_is_catkin_build (sawUniversalRobot_IS_CATKIN_BUILT)
-  if (sawUniversalRobot_IS_CATKIN_BUILT)
-    set (sawUniversalRobot_CONFIG_FILE_DIR
-         "${CATKIN_DEVEL_PREFIX}/share/sawUniversalRobot/cmake")
-    set (EXECUTABLE_OUTPUT_PATH "${CATKIN_DEVEL_PREFIX}/bin")
-    set (LIBRARY_OUTPUT_PATH    "${CATKIN_DEVEL_PREFIX}/lib")
-  else ()
-    if (NOT LIBRARY_OUTPUT_PATH)
-       set (LIBRARY_OUTPUT_PATH "${CMAKE_CURRENT_BINARY_DIR}/lib")
-    endif (NOT LIBRARY_OUTPUT_PATH)
-    set (sawUniversalRobot_CONFIG_FILE_DIR "${sawUniversalRobot_BINARY_DIR}")
-  endif ()
-
-  set (CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+  cisst_set_output_path ()
 
   # create/configure file for find_package (sawUniversalRobot)
-  set (sawUniversalRobot_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/include"
-                                     "${CMAKE_CURRENT_BINARY_DIR}/include")
+  set (sawUniversalRobot_INCLUDE_DIR
+    "${CMAKE_CURRENT_SOURCE_DIR}/include"
+    "${CMAKE_CURRENT_BINARY_DIR}/include")
```

Comment:
I don't think we create an "include" directory in the build tree any more

Reply:
The macro `cisst_add_config_files` creates a `<component>Revision.h` which is by default generated in <component>_BINARY_DIR/include by default.

Reply:
Ah, ok. I had deleted this file and it was not regenerated by the build, but that is because I did not rerun CMake.  I see it now.